src/search-scenario/azext_search_scenario/commands.py

```diff
@@ -0,0 +1,10 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def load_command_table(self, _):
+
+    with self.command_group('') as g:
+        g.custom_command('search-scenario', 'search_scenario')
```

Comment:
Did we discuss with PM about the command name?

Reply:
@chasewilson Hi, may I ask do you have any idea or suggestion for the command name?